Adds a tip about ERP and More (#9847)
Citadel-Station-13/Citadel-Station-13-6th-Port: strings/sillytips.txt

Maints.
BZ stops or slows down Lings chem regeneration drastically, make sure to BZ flood the station when lings are confirmed!
Admins always regret meme options in their polls.
ERP stands for Emergent Role Play.
Remember if you see a bug report it on Github.
Have good ideas for the game? Well make a suggestion on Github, we have templates just for it!
Role Play is fun. Try it sometimes.
This is a MRP Server, no really.
Clean bot has more to say after being emaged. Try it sometimes with your limited use Emags... It will be worth it!

> Bar sign can be emaged.

Reviewer: Pretty sure it would be spelled "emagged" but it's a made up word so who knows.

Author: e like the letter
mag like a /mag/net right?
so it should be 1 g?

Reviewer: pretty sure the past tense of every verb that ends in a G has two G's. Lugged, sagged, hugged, gagged, etc.

Mood.

> Ok Boomer, is not a helpful thing to say when being bowinked about your wrong doing.

Reviewer: Bwoinked, though I'm not sure about this one either.

Author: Bowinked, as its the sound of a bow like a cross/bow/, and a ink like pen and /ink/?

Reviewer: what

Maints.
BZ stops or slows down Lings chem regeneration drastically, make sure to BZ flood the station when lings are confirmed!
Admins always regret meme options in their polls.
ERP stands for Emergent Role Play.

> If your ever board, and don't want to hangout with people, just ask Sci for a Tetris board and make yourself usefull well not playing SS13!

Reviewer 2: ```suggestion
If you’re ever bored and don't want to hangout with people, just ask science for a Tetris board and make yourself useful while not playing SS13!
```
